test/configuration/karma-common.js

```diff
@@ -15,8 +15,7 @@ module.exports = {
     'test-settings.js',
 
     // The oc-client bundle
-    '../../vendor/l.js',
-    '../../src/oc-client.js',
+    '../../dist/oc-client.min.js',
```

Comment:
this is because l.js is now part of oc-client?

Reply:
ok, I see that it now reads from the /dist/ and that's the reason for this change